plugins/woocommerce/client/blocks/tests/e2e/tests/mini-cart/mini-cart-block.merchant.classic_theme.spec.ts

```diff
@@ -1,22 +1,12 @@
 /**
  * External dependencies
  */
-import {
-	expect,
-	test,
-	CLASSIC_THEME_SLUG,
-	BlockData,
-} from '@woocommerce/e2e-utils';
+import { expect, test, CLASSIC_THEME_SLUG } from '@woocommerce/e2e-utils';
 
-const blockData: BlockData = {
-	name: 'Mini-Cart',
-	slug: 'woocommerce/mini-cart',
-	mainClass: '.wc-block-minicart',
-	selectors: {
-		frontend: {},
-		editor: {},
-	},
-};
+/**
+ * Internal dependencies
+ */
+import { blockData } from './utils';
```

Comment:
Although not directly related to this PR, I took the opportunity to optimise this section while working in this area of the tests.